docs/api-testing/vault.md

```diff
@@ -36,21 +31,20 @@ The screenshot below shows the **Company Vault**; these variables are available
 
 <img src={useBaseUrl('img/api-fortress/2021/04/company_vault.png')} alt="Company Vault View"/>
 
-
-You should define a variable at the Project-specific **Vault** when you need to use the same one across multiple tests. As an example, you could save a password here as a variable and reuse it in multiple tests.
+You should define a variable at the project-specific **Vault** when you need to use the same one across multiple tests. As an example, you could save a password here as a variable and reuse it in multiple tests.
 
 :::tip Import Postman Collections into The Vault
 Additionally, you can import variables from Postman. See [here](/api-testing/import-postman-collection) for more details.
```

Comment:
Change "See [here]" to "See [Importing Postman Collections, Variables, and Environments]"